lib/membrane/aac/parser.ex

```diff
@@ -65,7 +68,11 @@ defmodule Membrane.AAC.Parser do
   end
 
   @impl true
-  def handle_stream_format(:input, _stream_format, _ctx, state) do
+  def handle_stream_format(:input, %Membrane.RemoteStream{} = _stream_format, _ctx, state) do
+    if state.in_encapsulation == :none and state.out_encapsulation == :ADTS do
+      raise "Not supported parser configuration: in_encapsulation: :none, out_encapsulation: :ADTS"
+    end
```

Comment:
As we discussed, would be nice to elaborate on this error i.e. why we can't support this - we lack some extra parameters etc.

Reply:
Totally agree, the error should explain why it is not supported and suggest a solution or at least point into the right direction

Reply:
done